tests/helpers_tests/test_condition.py

```diff
@@ -1381,10 +1362,8 @@ def test_exact__user_value__None(self):
 
         mock_log = getattr(self.mock_client_logger, log_level)
         mock_log.assert_called_once_with(
-            (
-                'Audience condition "{}" evaluated to UNKNOWN because a null value was passed for user attribute '
-                '"favorite_constellation".'
-            ).format(json.dumps(expected_condition_log))
+            f'Audience condition "{json.dumps(expected_condition_log)}" evaluated to UNKNOWN '
+            'because a null value was passed for user attribute "favorite_constellation".'
```

Comment:
This is all in one line correct? If two lines, will need backslash to break. Check?
also the corrections above and other ones....